app/assets/javascripts/components/overview/details.jsx

```diff
@@ -295,6 +299,22 @@ const Details = createReactClass({
       );
     }
 
+    if (this.props.editable) {
+      projectSelector = (
+        <HomeWikiProjectSelector
+          course={this.props.course}
+        />
+      );
+    }
+
+    if (this.props.editable) {
```

Comment:
These go together, so use the same `if` statement for both.

Also, these should not appear on the Wiki Ed dashboard, so add `!Features.wikiEd` to the conditional.